environments/dev/terraform/command-center.tf

```diff
@@ -19,8 +19,8 @@ module command_center {
   }
 
   is_production = false
-  k8s_cluster_size = 2
-  k8s_machine_type = "n1-standard-2"
+  k8s_cluster_size = 3
+  k8s_machine_type = "n1-standard-4"
```

Comment:
This was needed because the resource requests of parallel Argo steps + the new resource requests of Prometheus caused the workflow to deadlock.

NOTE: This caused a full cluster rebuild, which triggered some follow-up changes.

Reply:
Would it be beneficial at all to have these be inputs that are passed in as opposed to hard-coded values? Or would we prefer to have it be hard-coded since the command-center cluster will eventually not be running processes for specific project pipelines (or at least not the chonkier parts of them).

Reply:
They are variables in the command-center module, but they have to be hard-coded somewhere eventually 😄 this is that spot for now